make_queue/templates/make_queue/machine_card.html

```diff
@@ -33,7 +33,7 @@
     {% if machine.machine_type.has_stream %}
         <div id="fader"></div>
         <i id="closefullscreen" class="ui times icon"></i>
-        <script>var streamID = "stream-{% sanitize_stream_name machine %}"</script>
+        <script>let streamID = "stream-{% sanitize_stream_name machine %}";</script>
```

Comment:
This will be redefined for each stream, so `let` will result in a Javascript error. See the `/reservation/` page as an example.